src/input/touch.js

```diff
@@ -93,16 +93,17 @@ define(function (require) {
   function getTouchPos(canvas, e, i) {
     i = i || 0;
     var rect = canvas.getBoundingClientRect();
-    return {
-      x: e.changedTouches[i].clientX - rect.left,
-      y: e.changedTouches[i].clientY - rect.top
+    var touch = e.touches[i] || e.changedTouches[i];
+    return  {
+      x: touch.clientX - rect.left,
+      y: touch.clientY - rect.top
     };
   }
 
   /**
    * The touchStarted() function is called once after every time a touch is
    * registered. If no touchStarted() function is defined, the mousePressed()
-   * function will be called instead if it is defined. Browsers may have 
+   * function will be called instead if it is defined. Browsers may have
```

Comment:
My text editor removes empty spaces from the end of lines, that's what these (here to end of file) changes are.